app/helpers/application_helper.rb

```diff
@@ -28,7 +28,8 @@ def index_of_page object
   end
 
   def option_status_bill
-    Bill.statuses.map{|key, value| [t(".#{key}"), value]}
+    bill = Bill.statuses.map{|key, value| [t("global.#{key}"), value]}
+    bill.push([t("global.all"), 4])
```

Comment:
all thì truy vấn như thế nào e

Reply:
phần này là lúc mới vào trang thì em để load hết dữ liệu ra , sau khi em filter theo các trạng thái  em muốn quay về load hết toàn bộ dữ liệu thì e để thêm all vào cái select_tag ạ